Newsparser/ArticleParser.py

```diff
@@ -0,0 +1,29 @@
+from newspaper import Article
```

Comment:
looks dope. can you make a /cloud-functions top level directory and put this in there as ```newsparser` instead of `Newsparser`